doc: nrf: libraries: networking: cleanup cloud unlink section (#6178)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud.rst

Removing the link between device and user
*****************************************

If you want to remove the link between a device and an nRF Cloud account, you must do this from nRF Cloud.

> It is not possible for a device to remove itself from an nRF Cloud account.

Reviewer: ```suggestion
A device cannot remove itself from an nRF Cloud account.
```
